.pre-commit-config.yaml

```diff
@@ -21,7 +21,7 @@ repos:
     -   id: flake8
 
 -   repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.950
+    rev: v0.942
```

Comment:
This should be `v0.950` though?

Reply:
let me back track, my pycharm didn't grab the changes properly and sent me down a deep rabbit hole